examples/vllm_v1/configs/deepseek_r1/disagg_dp.yaml

```
@@ -0,0 +1,68 @@
+# depending on the DP configuration of the VllmPrefillWorker / VllmDecodeWorker
+VllmDpWorker:
+  # [NOTE] 'data_parallel_address' and 'data_parallel_start_rank' will be set differently
+  # depending on the DP worker counts and where the DP worker is located.
+  # See README.md for more details.
+  data_parallel_start_rank: 8
+  headless: true
+  ServiceArgs:
+    workers: 1
+    resources:
+      gpu: '8'
+  common-configs: [model, kv_transfer_config, served_model_name, data_parallel_size, data_parallel_size_local, data_parallel_address, data_parallel_rpc_port, max-model-len, trust-remote-code, enable_expert_parallel]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)